pkg/cmd/cli/install/install.go

```diff
@@ -347,6 +355,37 @@ func (o *InstallOptions) Run(c *cobra.Command, f client.Factory) error {
 	return nil
 }
 
+func (o *InstallOptions) CheckPluginImageRepo(f client.Factory) error {
+	clientset, err := f.KubeClient()
+	if err != nil {
+		errMsg := fmt.Sprint("Failed to get clientset.")
+		return errors.New(errMsg)
+	}
+	deployment, err := clientset.AppsV1().Deployments(o.Namespace).Get("velero", metav1.GetOptions{})
```

Comment:
Is Deployments("namespace") guaranteed to never return nil?  "velero" should be a constant.

Reply:
Ran test to check whether clientset.AppsV1().Deployments(o.Namespace) returns nil if no such deployment exit. clientset.AppsV1().Deployments(o.Namespace) will not return nil and clientset.AppsV1().Deployments(o.Namespace).Get("velero", metav1.GetOptions{}) will return error.